actionpack/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Rails 6.0.0.rc1 (April 24, 2019) ##
 
+*   Fix a bug where DebugExceptions errors out when malformed query parameters are provided
+
+    *Yuki Nishijima*, *Stan Lo*
+
```

Comment:
This should be above the title header since this change will be for 6.0.0.rc2 not 6.0.0.rc1